py/makeqstrdata.py

```diff
@@ -220,6 +220,66 @@
     "values",
     "write",
     "zip",
+    # Additional QSTRs that must have index <255 (these are not part of
+    # the .mpy compatibility list though).
+    "__bool__",
```

Comment:
Why are these added here, don't they increase the size of ports that don't use these qstrs?  Eg bare-arm and minimal.

Reply:
Because these qstrs must have index <255 (e.g. the operator code assumes that the qstr for the operator name is a byte). Therefore they cannot go into a sorted pool, so they must be put in the unsorted pool at the start.

makeqstrdata.py only append these here _if_ they appear in the required qstr list from the firmware.

Reply:
But in a comment you made above you said that:

> It adds a bunch of fixed qstrs (mostly dunder operators) that were not previously included in the bare-arm build but now need to be forced into the first pool so that their id < 256. 

So... is it possible to not include these in the build if they aren't used?  (Mainly I'm interested in getting the bare-arm size diff down.)